src/process-config.ts

```diff
@@ -1,10 +1,11 @@
 export function processConfig (config: any = {}, args: any = {}) {
   const username = config.username || process.env.SAUCE_USERNAME;
   const accessKey = config.accessKey || process.env.SAUCE_ACCESS_KEY;
+  const region = config.region || process.env.SAUCE_REGION;
   const startConnect = config.startConnect !== false;
 
   let tunnelIdentifier = args.tunnelIdentifier || config.tunnelIdentifier;
-  let seleniumHostUrl = 'ondemand.saucelabs.com:80/wd/hub';
+  let seleniumHostUrl = region === 'eu' ? 'ondemand.eu-central-1.saucelabs.com:80/wd/hub' : 'ondemand.saucelabs.com:80/wd/hub';
```

Comment:
I would write it this way, a bit shorter:
```suggestion
  let seleniumHostUrl = `ondemand.${region === 'eu' ? 'eu-central-1' : ''}.saucelabs.com:80/wd/hub';
```